jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamo/Attributes.kt

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamo
```

Comment:
Can you add a link to the docs where the Dynamo data types are defined?

Reply:
done